package.json

```diff
@@ -32,7 +33,7 @@
   "dependencies": {
     "es6-collections": "0.x",
     "skatejs-named-slots": "1.x",
-    "webcomponents.js": "webcomponents/webcomponentsjs#b77ca74"
```

Comment:
That fork has a pull that is for implementing customised built-in element support. The best thing might be to try and merge in from the mainline into our v1 branch and release from that sha. I have a separate branch in skate testing webreflection/document-register-element instead of webcomponentsjs. If that works, then it might be a better alternative as we can pin to a version.